src/scss/components/banner/_banner--fullpill.scss

```diff
@@ -0,0 +1,152 @@
+@charset "UTF-8";
+
+$banner-circle--image-width: 200px;
+$banner-circle--sm-image-width: 150px;
+
+
+.chemh-wrapper--banner-fullpill {
+  max-width: calc(100vw - 200px);
```

Comment:
It looks like this max-width constraint is causing some issues for mobile, can you address that?
![Screenshot 2023-09-21 at 12 33 58 PM](https://github.com/SU-SWS/chem_h_subtheme/assets/49299083/abb44fdd-6204-4a64-bdbf-56ecbbb1dc88)